API/chat/forms.py

```diff
@@ -32,23 +32,12 @@ def save(self):
 
 
 class MessagePatchForm(forms.Form):
-    id = forms.IntegerField()
     text = forms.CharField(widget=forms.Textarea)
     datetime_sent = forms.IntegerField()
     typing = forms.BooleanField(required=False)
 
-    def is_valid(self):
-        if not super(MessagePatchForm, self).is_valid():
-            return False
-
-        message = Message.objects.get(pk=self.cleaned_data['id'])
-        if not message.typing:
-            return False
-
-        return True
-
     def save(self):
-        message = Message.objects.get(pk=self.cleaned_data['id'])
+        message = self.message
```

Comment:
@Geekfish can comment on whether this is good practice, but I think each form in Python should have its attributes defined in the class definition?